AndroidAnnotations/androidannotations-core/androidannotations/pom.xml

```diff
@@ -56,6 +56,11 @@
 			<version>4.1.1.4</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>1.10.8</version>
```

Comment:
Moreover, we use Mockito in other modules as well. So you can pull it to the root POM in `dependencyManagement`, just like we do with JUnit.